week1/page-2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Make Your Home Great Again,Home Office</title>
+    <meta name="author" content="Ekip Kalir">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="main-2.css">
+    <script src="https://kit.fontawesome.com/f32ec08315.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="nav-bar-top">
+        <header>
+            <ul>
+                <li><a href="#company-details">BlogX</a></li>
+                <li><a href="#articles-whole">Content</a></li>
+                <li><a href="#contact-details">Contact</a></li>
+                <li style="float:right"><a class="active" href="#summary">About</a></li>
+            </ul>
+        </header>
+    </div>
+    <br>
+    <br>
+    <section class="statistics-social-media">
+        <div class="form-social-media">
+            <label>Please select your three favorite home office and social media platform</label>
+            <label>
+                <input type="checkbox" name="status">
+                Facebook
+            </label>
+            <label>
+                <input type="checkbox" name="status">
+                Slack
+            </label>
+            <label>
+                <input type="checkbox" name="status">
+                Youtube
+            </label>
+            <label>
+                <input type="checkbox" name="status">
+                Zoom
+            </label>
+            <label>
+                <input type="checkbox" name="status">
+                Hangouts
+            </label>
+            <label>
+                <input type="checkbox" name="status">
+                Whatsupp
```

Comment:
not much, whatsupp with you?

Reply:
Sorry, after the page-1 index.html, I wanted to continue with page-2 but could not finish. I can delete, to be honest.